src/checkout/SuccessOverlay.tsx

```diff
@@ -0,0 +1,41 @@
+import {Overlay} from '@influxdata/clockface'
+import React, {FC, useContext} from 'react'
+import {CheckoutContext} from 'src/checkout/context/checkout'
+import {RemoteDataState} from 'src/types'
+
+const SuccessOverlay: FC = () => {
+  const {checkoutStatus, onSuccessUrl} = useContext(CheckoutContext)
+  const handleClick = () => (window.location.href = onSuccessUrl)
```

Comment:
move this to a history.push with react router. it helps the customer save resources and prevents us from sweeping state lifecycle issues under the rug